runtime/compiler/control/CompilationThread.cpp

```diff
@@ -770,7 +770,7 @@ TR::CompilationInfo::createCompilationInfo(J9JITConfig * jitConfig)
          that we do memset this object to 0 */
       memset(alloc, 0, sizeof(TR::CompilationInfo));
       _compilationRuntime = new (alloc) TR::CompilationInfo(jitConfig);
-
+      J9JITConfig::compilationRuntime = *_compilationRuntime;
```

Comment:
You cannot use this syntax because the new field is not a static.
Use `jitConfig->compilationRuntime` instead (plus any casts required to convert to void*)
Also, `_compilationRuntime` is already a pointer, so you don't have to use the `*` in front of it.